frontend/conf/routes

```diff
@@ -61,18 +61,17 @@ GET         /masterclasses/:tag/:subTag                controllers.WhatsOn.maste
 GET         /event/:id                                 controllers.Event.details(id)
 GET         /event/:id/embed                           controllers.Event.embedData(id)
 GET         /event/:id/buy                             controllers.Event.buy(id)
-GET         /event/:id/thankyou                        controllers.Event.thankyou(id, oid: Option[String])
```

Comment:
this is no longer used, I grepped 5 days of fastly logs and only bots had hit it